interfaces/builtin/mpris.go

```diff
@@ -221,14 +224,40 @@ func (iface *MprisInterface) ConnectedSlotSnippet(plug *interfaces.Plug, slot *i
 	}
 }
 
+func (iface *MprisInterface) GetName(attribs map[string]interface{}) (string, error) {
+	// default to snap name if 'name' attribute not set
+	mprisName := "@{SNAP_NAME}"
+	for attr := range attribs {
```

Comment:
I had this discussion early on when interfaces were still called capabilities. We decided to ignore unknown attributes and allow any attributes to exist.